src/tests/suite/daos_rebuild.c

```diff
@@ -767,7 +770,7 @@ rebuild_destroy_container(void **state)
 {
 	test_arg_t	*arg = *state;
 	test_arg_t	*args[2] = { 0 };
-	daos_obj_id_t	oids[OBJ_NR * 100];
+	daos_obj_id_t	oids[OBJ_NR];
```

Comment:
I'm OK to decrease the number of objects by default, but can we have an option to run larger object numbers?